README.md

```diff
@@ -51,13 +51,13 @@ To set up your development environment,
 # Build
 IntelliJ will automatically build your code as needed.  To build and unit test using Gradle, run:
 ```shell
-./gradlew build
+./gradlew <project-name> build
```

Comment:
Do we need to specify the project name here?  Leaving it out feels like a simpler starting point for someone first reading about the project - maybe have a separate line/section showing how to build everything vs building a specific sub-project?

Also, should it be `./gradlew :<project-name>:build`?

Reply:
But it still needs the colon between the project name and the task name doesn't it?